remme/tp/node_account.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+
+from remme.protos.account_pb2 import AccountMethod
+from remme.protos.node_account_pb2 import (
```

Comment:
Not this

```
from .. import (
    .., ..,
)
```

but this

```
from .. import (
    .., 
    ..,
)
```

Remember, 

> when you get some point, find all occurrences in the pull request and apply.

Reply:
Done.